proxy/api/src/http/project.rs

```diff
@@ -210,6 +223,15 @@ mod handler {
 
         Ok(reply::json(&feed))
     }
+
+    /// Track the peer for the provided project.
+    pub async fn track(
+        ctx: context::Context,
+        super::TrackInput { urn, peer_id }: super::TrackInput,
```

Comment:
Just a suggestion: Import `TrackInput` from `super` and simplify to:

```suggestion
        TrackInput { urn, peer_id }: TrackInput,
```

TIL btw :smiley:

Reply:
This suggestion goes against the way things are organised in the http handlers in general. I'm operating under the "When in Rome" principle and adhering to the style that currently exists :)

Reply:
macht Sinn!